code/modules/fish/fish_tank.dm

```diff
@@ -263,12 +266,12 @@
 	handle_special_interactions()
 
 	//Handle water leakage from damage
-	if(leaking && leaking)
```

Comment:
This was supposed to be fixed by #17227, not sure why it doesn't conflict.

Reply:
Because you're looking at deletions and I started this branch before that PR

I actually opened that Issue while doing the research for this branch. As I was aware of the changes you made, I simply mirrored them exactly so it wouldn't conflict.